publishers/community/generic.py

```diff
@@ -39,3 +42,83 @@ def remove_internal_fields(_, publication):
     publication.pop('outputs', None)
 
     return publication
+
+
+@Register
+def remove_streamalert_normalization(_, publication):
+    """This publisher removes the super heavyweight 'streamalert:normalization' fields"""
+
+    # Python is bad at recursion so I managed to tip toe around that with BFS using a queue.
+    # This heavily takes advantage of internal references being maintained properly as the loop
+    # does not actually track the "current scope" of the next_item.
+    fringe = deque()
+    fringe.append(publication)
+    while len(fringe) > 0:
+        next_item = fringe.popleft()
+
+        if isinstance(next_item, dict):
+            if Normalizer.NORMALIZATION_KEY in next_item.keys():
```

Comment:
I actually used this intentionally because during the iteration, I can remove some of the keys. If I use `if key in next_item:` I get the "dictionary size changed during iteration" error.